clients/js/test/data/1.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
```

Comment:
Can we use the filename as one instead of numeric one?

Reply:
this I will change.